src/counters/timer.rs

```diff
@@ -37,15 +38,15 @@ impl Timer {
     pub fn start(&mut self) {
         if self.enabled {
             self.time = 0.0;
-            self.start = Some(instant::now());
+            self.start = Some(Instant::now().elapsed().as_secs_f64());
```

Comment:
These are incorrect. `Instant::now().elapsed().as_secs_f64()` will always return something roughly equal to zero (it measures the time between the call to `Instant::now()` and the call to `.elapsed()`).
I think the right approach here is to replace `Timer::start: Option<f64>` by a `Option<Instant>`.
```suggestion
            self.start = Some(Instant::now());
```